### PR TITLE
Orient 635 Abbadie 90

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Some minor corrections.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient635.xml [hunk 1]
```diff
@@ -236,9 +235,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                     <history>
                         <origin>
-                            <origDate notBefore="1650" notAfter="1699" evidence="lettering"/>
+                            <origDate notBefore="1680" notAfter="1611" evidence="internal-date">Shortly before 1711.</origDate>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient635.xml:238]: can you explain where in the manuscript the date is given? I didn't find it, reading the file only quickly

File: ParisBNF/abb/BNFabb90.xml [hunk 2]
```diff
@@ -105,12 +105,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                           </objectDesc>
                         <handDesc>
                            <handNote xml:id="h1" script="Ethiopic">
-                              <seg type="script">16th century.</seg>
+                               <locus from="1rb" to="31va"/>                               
+                               <locus from="57ra" to="71vb"/>
+                               <locus from="81ra" to="88ra"/>
+                              <seg type="script">late 15th/first half of the 16th century.</seg>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb90.xml:111]: dates, here and below

**Review by thea-m — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb90.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb90.xml:111]: What is about them?

File: LondonBritishLibrary/orient/BLorient635.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply LondonBritishLibrary/orient/BLorient635.xml:238]: There are no dates, there is only the name of King Tewoflos, written in the same hand as the main text. He ruled 1708-1711, the manuscript was probably written shortly / some time before that.

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**